### ISSUE TITLE
Data prior to 2006

### ISSUE THREAD

**Issue by zmeers** [openaustralia/publicwhip#1184]
Would it be possible to scrape voting data prior to 2006? I'm looking to go back to 1950. I'm happy to build a parser myself but I assume there is a reason it hasn't been done... Is it possible to grab data prior to 2006?

**Comment by jamezpolley** [comment]
I've only started working with these tools recently so there might be other issues I'm not aware of, but the first thing that occurs to me is that you'd have to extend the csv data files back until 1997.  The name parsing code is also quite focused on parsing the data we've hard to parse; I would bet that as you extend it back in time you're going to find more special cases that need to be added.

But in theory, assuming the format didn't change, I can't see why it wouldn't work.

FWIW it looks like you can get at least Senate data going all the way back to a discusson on MORTALITY OF KANAKAS in http://parlinfo.aph.gov.au/parlInfo/download/hansard80/hansards80/1901-12-13/toc_unixml/19011213_senate_1_7.xml;fileType=text%2Fxml - http://parlinfo.aph.gov.au/parlInfo/search/summary/summary.w3p;adv=yes;orderBy=_fragment_number,doc_date-rev;query=Dataset%3Ahansards,hansards80%20Decade%3A%221900s%22%20Year%3A%221901%22;resCount=Default

**Comment by zmeers** [comment]
Yep, that makes sense. I'll take a stab at parsing the data this weekend.

**Comment by cofiem** [comment]
@zmeers @jamezpolley Did you end up giving this a go? I could give this a go and/or assist if you've already started.
